.github/workflows/release-api.yml

```
@@ -2,8 +2,10 @@ name: Publish API image to Hub
 
 on:
   push:
+    branches: 
+      - main
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+$'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

What if we specify the exact pattern 

```suggestion
      - 'v[0-9]+.[0-9]+.[0-9]+$'
```

---

Github Actions filter pattern does not seem to be regex https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet